docs/team/adding-members.rst

```diff
@@ -61,6 +61,7 @@ issue and the team member is now ready to go!
    - [ ] A gmail account (for GKE credentials + Google team drive)
    - [ ] A [PyPI account](https://pypi.org) (for PyPI access)
    - [ ] A [Discourse account](https://discourse.jupyter.org)
+   - [ ] A [Quay.io account](https://quay.io/)
```

Comment:
I'm undecided on whether it's *needed* or useful. From a security perspective it's better not to hand out admin access to everything by default to people who are unlikely to need it, especially when the org is large enough that there's enough redundancy amongst the existing account holders. I'd consider making this optional e.g. `Things a new team member can have/may want`, but this is not a blocker for this PR, it can be discussed later.

Reply:
Fair enough - I agree with your sentiment here...though we don't have a policy for when a person should get access. I feel like we have 3 options:

- design that policy here
- assume people get access by default when they're added to the team
- remove from this list and tell people to ask once it's relevant

I'm fine w/ option 3 for now, will remove this line in case you agree

Reply:
I think quay.io / PyPI are in the same boat - not everyone *needs* access to these things, especially as releases are ~all published from CI. But we do need to make sure that *enough* people have access to them that we don't end up blocking on one/few folks for the infrequent but usually important admin tasks that need these accounts.

Maybe these should be explicit teams?